package.json

```diff
@@ -114,9 +119,10 @@
         "js-yaml": "^3.13.1",
         "jsonpath-plus": "^3.0.0",
         "mustache": "3.0.1",
+        "npm": "^6.14.5",
```

Comment:
why do we need npm as a dependency?

Reply:
It was added automatically, will remove it.